File: ios/AlpineApp/AlpineApp/Models/GroupModels.swift
import Foundation

struct GroupInfo: Codable, Identifiable, Sendable {
    let groupId: Int64
    let groupName: String
    let description: String
    let numPeers: Int64
    let totalQueries: Int64
    let totalResponses: Int64

    var id: Int64 { groupId }
}

File: android/AlpineApp/app/src/main/java/com/alpine/app/data/model/GroupModels.kt
package com.alpine.app.data.model

import kotlinx.serialization.Serializable

@Serializable
data class GroupInfo(
    val groupId: Long,
    val groupName: String,
    val description: String,
    val numPeers: Long,
    val totalQueries: Long,
    val totalResponses: Long
)
